hadoop-hdds/docs/content/interface/S3-Tenant.md

```diff
@@ -0,0 +1,324 @@
+---
+title: "Tenant commands"
+menu:
+   main:
+      parent: "Command Line Interface"
```

Comment:
I still think this would fit better as a subsection under the multi-tenancy header. I think it disrupts the flow having the docs split under two separate headers. Also, pretty much every feature adds new CLIs but they put them with their feature, not in this section, so I think we should do the same.